File: backend-v2/src/app/models/Beacon.js
const { Model } = require('sequelize');
const Sequelize = require('sequelize');

class Beacon extends Model {
  static init(sequelize) {
    super.init(
      {
        id: {
          type: Sequelize.STRING,
          primaryKey: true,
        },
        content_name: Sequelize.STRING,
        content_type: Sequelize.STRING,
        content: Sequelize.STRING,
        content_description: Sequelize.STRING,
      },
      {
        sequelize,
      }
    );

    return this;
  }
}

module.exports = Beacon;

File: Mobile/lib/models/Beacon.dart
class Beacon {
  String content_type;
  String content;
  String content_description;
  String id;
  String content_name;

  Beacon.fromJson(Map<String, dynamic> json) {
    content_type = json['content_type'];

    content = json['content'];
    content_description = json['content_description'];
    content_name = json['content_name'];
    id = json['id'];
  }
  Map<String, dynamic> toJson() => {
      'content_type': this.content_type,
      'content': this.content,
      'content_description': this.content_description,
      'id': this.id,
      'content_name': this.content_name
    };
}